app/src/main/java/hr/dice/filipbionda/tmdbpractice/ui/showscreen/ShowScreen.kt

```diff
@@ -0,0 +1,671 @@
+import hr.dice.filipbionda.tmdbpractice.R
+import hr.dice.filipbionda.tmdbpractice.data.mocks.mockMovies
+import hr.dice.filipbionda.tmdbpractice.data.models.Actor
+import hr.dice.filipbionda.tmdbpractice.data.models.MediaItem
+import hr.dice.filipbionda.tmdbpractice.ui.components.ExpandedButton
+import hr.dice.filipbionda.tmdbpractice.ui.theme.TMDBPracticeTheme
+import hr.dice.filipbionda.tmdbpractice.ui.theme.white_CC
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+
+val mockMovie = MediaItem(
+    title = "Inception",
+    rating = 87,
+    coverPath = "https://im.jdmagicbox.com/comp/jd_social/news/2021dec11/image-1092864-84qgkwd5wz.jpg?clr=#382e38",
+    date = "21/07/2022",
+    language = "EN",
+    overview = "A thief who steals corporate secrets through the use of dream-sharing technology is given the inverse task of planting an idea into the mind of a C.E.O.",
+    actors = listOf(
+        Actor(name = "Leonardo", surname = "DiCaprio" ,imagePath = "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcS8yhDDv3b5ixR0-eeKOz9sfth6qlJxUFRuMk3fRJktRUVlO4fJ"),
+        Actor(name = "Joseph", surname = "Gordon-Levitt", imagePath = "https://m.media-amazon.com/images/M/MV5BMTY3NTk0NDI3Ml5BMl5BanBnXkFtZTgwNDA3NjY0MjE@._V1_.jpg"),
+        Actor(name = "Ellen", surname = "Page", imagePath = "https://resizing.flixster.com/VsatQP6O9THxjHuKRsWwXjtPBAg=/fit-in/352x330/v2/http://media.baselineresearch.com/images/1260564/1260564_full.jpg")
+    ),
+    categories = listOf("Action", "Sci-Fi", "Thriller", "Anime", "Sports", "Specials"),
+    duration = 140,
+    seasons = null,
+    trailerUrl = ""
+)
+
+@Composable
+fun ShowScreen(
+    mediaItem: MediaItem,
+    navigateBack: () -> Unit,
+    openRecommendedMediaItem: (MediaItem) -> Unit,
+    playTrailer: (String) -> Unit ,
+    modifier: Modifier = Modifier
+){
+    Box(
+        modifier = modifier
+            .fillMaxSize()
+            .background(color = MaterialTheme.colorScheme.surfaceContainer)
+    ){
+        val columnState = rememberScrollState()
+        ShowScreenAppBar(
+            navigateBack = navigateBack,
+            addToFavorites = {
+                // TODO - cache MediaItem
+            },
+            modifier = Modifier
+                .offset(y = dimensionResource(R.dimen.app_bar_y_offset))
+                .padding(end = dimensionResource(R.dimen.show_screen_horizontal_padding))
+                .zIndex(1f)
+                .fillMaxWidth()
+        )
+        Column(
+            horizontalAlignment = Alignment.Start,
+            modifier = Modifier
+                .fillMaxSize()
+                .background(color = MaterialTheme.colorScheme.surfaceContainer)
+                .verticalScroll(
+                    state = columnState,
+                )
+        ) {
+            Box(
+                modifier = Modifier.fillMaxWidth(),
+                contentAlignment = Alignment.Center
+
+            ) {
+                ShowScreenCoverImage(
```

Comment:
Composable functions names should be nouns and not verbs if they don't have a return value